verticapy/vdataframe.py

```diff
@@ -4484,55 +4340,17 @@ def chaid_columns(self, columns: list = [], max_cardinality: int = 16):
     def copy(self):
         """
     ---------------------------------------------------------------------------
-    Returns a copy of the vDataFrame.
+    Returns a deep copy of the vDataFrame.
 
     Returns
     -------
     vDataFrame
         The copy of the vDataFrame.
         """
-        copy_vDataFrame = vDataFrame("", empty=True)
```

Comment:
please double check if you want to delete this.

Reply:
yes deep copy is exactly what I want.